Lib/ufo2ft/featureWriters/markFeatureWriter.py

```diff
@@ -239,8 +339,7 @@ class MarkFeatureWriter(BaseFeatureWriter):
 
     # Glyphs moves "_bottom" and "_top" (if present) to the top of
     # the list and then picks the first to use in the mark feature.
-    # https://github.com/googlei18n/noto-source/issues/122
-    # #issuecomment-403952188
+    # https://github.com/googlei18n/noto-source/issues/122#issuecomment-403952188
     anchorSortKey = {"_bottom": -2, "_top": -1}
```

Comment:
this `anchorSortKey` doesn't seem to be used any more.

Reply:
what's the order now? You said that the last lookup wins. Should we not place `_bottom` and `_top` last so that they wins and we don't change existing fonts that may rely on this "feature"? Maybe we can sort alphabetically (like khaled is suggesting) but do an exception for those two.

Reply:
I implemented what you suggested above.